### PR TITLE
[15.0][MIG] web_archive_date

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

Depends on:
- https://github.com/OCA/server-ux/pull/685

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Hi @JasminSForgeFlow , the base module is merged. Can you rebase?

**Comment by JasminSForgeFlow** [comment]
> Hi @JasminSForgeFlow , the base module is merged. Can you rebase?

Its done already, Thanks

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
I expected to see the archive date and user in the view metadata info of an archive record, but it is missing:

**Comment by JasminSForgeFlow** [comment]
> I expected to see the archive date and user in the view metadata info of an archive record, but it is missing:
> 
> 

Its Fixed, Thanks

**Review by LoisRForgeFlow — COMMENTED** [review]
Could you cherry-pick the fix done here https://github.com/OCA/server-ux/pull/687?

Also, your commit history only contains the pre-commit commit, you need to have the pre-commit and migrations changes separated in [2 different commits](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#technical-method-to-migrate-a-module-from-140-to-150-branch) to ease review. Could you do that?

**Comment by JasminSForgeFlow** [comment]
> Could you cherry-pick the fix done here #687?
> 
> Also, your commit history only contains the pre-commit commit, you need to have the pre-commit and migrations changes separated in [2 different commits](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#technical-method-to-migrate-a-module-from-140-to-150-branch) to ease review. Could you do that?

https://github.com/OCA/server-ux/pull/687 included and commits also fixed.
Thanks

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

File: web_archive_date/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Format of `archive_date` is not the same as `write_date` shown in the metadata. Values are formatted in javascript. Can't inherit the method and adapt the value there.
+
+TO BE DONE: Check how to format the value in the QWeb view somehow.
```
Comment (GuillemCForgeFlow) [inline web_archive_date/readme/ROADMAP.rst:3]:
```suggestion
```
You can remove the file. Thank you @JasminSForgeFlow for the Javascript changes! 👍🏽

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! 👍🏽

File: web_archive_date/readme/ROADMAP.rst (hunk 1 above)
Comment (JasminSForgeFlow) [reply web_archive_date/readme/ROADMAP.rst:3]: Updated, Thanks

**Pull request merged**